plugins/palo_alto_cortex_xdr/icon_palo_alto_cortex_xdr/util/util.py

```diff
@@ -9,8 +12,12 @@ def now_ms():
 
     @staticmethod
     def send_items_to_platform_for_trigger(
-        trigger, items, output_type, last_event_processed_time_ms, time_field="creation_time"
-    ):
+        trigger: Trigger,
+        items: List[Dict],
+        output_type: str,
+        last_event_processed_time_ms: int,
+        time_field: str = "creation_time",
+    ) -> Iterable[int]:
```

Comment:
```suggestion
    ) -> [int]:
```
Should be good enough